compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/input/pointer/SuspendingPointerInputFilter.kt

```diff
@@ -584,21 +584,21 @@ internal class SuspendingPointerInputModifierNodeImpl(
         if (lastEvent.changes.fastAll { !it.pressed }) {
             return // There aren't any pressed pointers, so we don't need to send any events.
         }
-        val newChanges = lastEvent.changes.fastMapNotNull { old ->
-            PointerInputChange(
-                id = old.id,
-                position = old.position,
-                uptimeMillis = old.uptimeMillis,
-                pressed = false,
-                pressure = old.pressure,
-                previousPosition = old.position,
-                previousUptimeMillis = old.uptimeMillis,
-                previousPressed = old.pressed,
-                isInitiallyConsumed = old.pressed
-            )
-        }
-
-        if (newChanges.isEmpty()) return
+        val newChanges =
+            lastEvent.changes.fastMapNotNull { old ->
+                PointerInputChange(
+                    id = old.id,
+                    position = old.position,
+                    uptimeMillis = old.uptimeMillis,
+                    pressed = false,
+                    pressure = old.pressure,
+                    previousPosition = old.position,
+                    previousUptimeMillis = old.uptimeMillis,
+                    previousPressed = old.pressed,
+                    isInitiallyConsumed = old.pressed,
+                    type = old.type
+                )
+            }
```

Comment:
Reformatting like this makes it a bit harder to review and a bit harder to look at `git blame` later.

Reply:
No need to discuss; if it's enforced by the aosp linter then it should be the same on our repo.